Fri May  9 14:51:31 2014: ==================================================
Fri May  9 14:51:31 2014: === Starting Radiative-convective model server ===
Fri May  9 14:51:31 2014: ===                                               
Fri May  9 14:51:31 2014: === Using 5 threads for RC model                 
Fri May  9 14:51:31 2014: === On 6 CPUs                                    
Fri May  9 14:51:31 2014: ===                                               
Fri May  9 14:51:31 2014: === Max simulation time: 300 seconds               
Fri May  9 14:51:31 2014: ==================================================
Traceback (most recent call last):
  File "run_rc_model.py", line 1072, in <module>
    runserver()
  File "run_rc_model.py", line 1067, in runserver
    srv = make_server(host, PORT, application)
  File "/usr/lib/python2.7/wsgiref/simple_server.py", line 144, in make_server
    server = server_class((host, port), handler_class)
  File "/usr/lib/python2.7/SocketServer.py", line 408, in __init__
    self.server_bind()
  File "/usr/lib/python2.7/wsgiref/simple_server.py", line 48, in server_bind
    HTTPServer.server_bind(self)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 108, in server_bind
    SocketServer.TCPServer.server_bind(self)
  File "/usr/lib/python2.7/SocketServer.py", line 419, in server_bind
    self.socket.bind(self.server_address)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
